tests/compiles/staticlibtest.nim

```diff
@@ -0,0 +1 @@
+echo 1234
```

Comment:
usually, convention is mfoo for a test foo; 
another option is to get rid of this and make tstaticlib.nim self-contained, either via this approach:
* tests/system/tsigexitcode.nim via `getAppFilename`
* or via an approach similar to `trunner`

Reply:
I renamed the file.